plugin/karma.c

```diff
@@ -228,6 +251,8 @@ static void karma_handler(cbot_event_t event, cbot_actions_t actions)
       actions.send(event.bot, event.channel, "sorry, you're not authorized to do that!");
     }
     recapfree(captures);
+  } else if (reexec(forget, event.message, NULL) != -1) {
```

Comment:
It could be done, but the fact is that it functions as both a boolean telling whether the message is addressed to the bot, as well as an increment to the rest of the message.  So either way you name it, it's missing one of its purposes.  So I would probably leave it.